msg/sees_manual_control_data.msg

```diff
@@ -0,0 +1,7 @@
+uint64 timestamp                        	# time since system start (microseconds)
+
+uint8 sees_desired_control_source		# desired type of manual control source, RC (1) or Mav (2)
+uint64 valid_mavlink_setpoint_count		# number of Mavlink connections providing setpoints (implying joystick connected). Should be < 2.
```

Comment:
Do we need `uint64` if the expected count is less than 2? I think `uint8`, similar to `sees_desired_control_source`, should be sufficient.